lib/requester/util.js

```diff
@@ -116,15 +130,23 @@ module.exports = {
      *
      * @todo: Move this to the SDK.
      * @param request
+     * @param options
      */
-    getRequestBody: function (request) {
-        var options,
-            mode = _.get(request, 'body.mode'),
+    getRequestBody: function (request, options) {
+        var mode = _.get(request, 'body.mode'),
             body = _.get(request, 'body'),
-            empty = request.body && request.body.isEmpty && request.body.isEmpty(),
-            content;
+            empty = body ? body.isEmpty() : true,
```

Comment:
Nothing will break, the extra guard was added back when we were maintaining compatibilty with an older version of the SDK which did not have this method :)